src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+public class Ball {
+    private String color;
+    private int number;
+    public void setColor(String color) {
+        this.color = color;
+    }
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+    public Ball() {
+    }
```

Comment:
The empty constructor is not being used and could be removed if you don't have a specific use case for creating a Ball object without setting its initial state.